napari/_vispy/camera.py

```diff
@@ -176,6 +181,21 @@ def _on_center_change(self):
     def _on_zoom_change(self):
         self.zoom = self._camera.zoom
 
+    def _on_orientation_change(self):
+        orientation_xyz = self._camera.orientation[::-1]
```

Comment:
Is this reorienting from xyz to zyx? If so adding a 1 line docstring would give context.